src/lib/Libattr/master_queue_attr_def.xml

```diff
@@ -404,6 +404,23 @@
 	<ECL>NULL_VERIFY_VALUE_FUNC</ECL>
 	</member_verify_function>
     </attributes>
+    <attributes>
+	/* QA_ATR_Comment */
+	<member_name><both>ATTR_comment </both></member_name>			<!-- "comment" --> 
+	<member_at_decode>decode_str </member_at_decode>
+	<member_at_encode>encode_str </member_at_encode>
+	<member_at_set>set_str </member_at_set>
+	<member_at_comp>comp_str </member_at_comp>
+	<member_at_free>free_str </member_at_free>
+	<member_at_action>NULL_FUNC </member_at_action>
+	<member_at_flags><both>NO_USER_SET | ATR_DFLAG_NOSAVM </both></member_at_flags>
```

Comment:
The consistency with the node comment is the reason why I use "NO_USER_SET | ATR_DFLAG_NOSAVM". I think it is the same case as the node comment. ATR_DFLAG_NOSAVM is not necessary but I suggest to keep it consistent. Now the admin, operator and manager are able to set the queue comment, which I think is correct.